test/v2/unit/coins/abstractUtxoCoin.js

```diff
@@ -117,5 +118,70 @@ describe('Abstract UTXO Coin:', () => {
     }));
 
 
+  });
+
+  describe('Recover Wallet:', () => {
+
+    let coin, bitgo;
+
+    before(() => {
+      bitgo = new TestBitGo({ env: 'mock' });
+      coin = bitgo.coin('tbtc');
+    });
+
+
+    it('should construct a recovery transaction', co(function *() {
+
+      // Initialize constants from a real testnet bitgo wallet that once contained real tbtc unspents
+      const userKey = '{"iv":"OVZx6VlJtv74kyE9gi5c0A==","v":1,"iter":10000,"ks":256,"ts":64,"mode":"ccm","adata":"","cipher":"aes","salt":"gY6e6MieSZ4=","ct":"O64y1HhJWxbST1 /KfiRXpSDBl3/d+Grphpq9IhWrXKI2m/V0H1fxRQPj4KCoCV0veEUAvvgSfi49vksEZ0PdXI66umlqWnTahqyQgddBiT05E8yB3HWzVBvwIoMfkL9acQhnL7phjwupZRy73EzeGEX9burWx3w="}';
```

Comment:
I think this is a great opportunity to do something I've been meaning to do for a while. Could you please move all these large, hard-coded strings to a new file in test/fixtures (this dir should be created)? I'd like to stop putting hard coded data like this in the test files directly.